tonic-build/src/server.rs

```diff
@@ -188,18 +207,31 @@ fn generate_trait<T: Service>(
 
 fn generate_trait_methods<T: Service>(
     service: &T,
+    emit_package: bool,
     proto_path: &str,
     compile_well_known_types: bool,
+    disable_comments: &HashSet<String>,
 ) -> TokenStream {
     let mut stream = TokenStream::new();
 
+    let package = if emit_package { service.package() } else { "" };
     for method in service.methods() {
         let name = quote::format_ident!("{}", method.name());
 
         let (req_message, res_message) =
             method.request_response_name(proto_path, compile_well_known_types);
 
-        let method_doc = generate_doc_comments(method.comment());
+        let method_doc = if disable_comments.contains(&format!(
+            "{}{}{}.{}",
+            package,
+            if package.is_empty() { "" } else { "." },
+            service.identifier(),
+            method.identifier()
```

Comment:
For a follow up PR if you have time would be good to move these to a fn to centralize the impl

Reply:
I have created a draft PR. https://github.com/hyperium/tonic/pull/1144